eng/common/scripts/Invoke-DevOpsAPI.ps1

```diff
@@ -2,6 +2,24 @@
 
 $DevOpsAPIBaseURI = "https://dev.azure.com/{0}/{1}/_apis/{2}/{3}?{4}api-version=6.0"
 
+function Get-Base64EncodedToken([string]$AuthToken)
+{
+  $unencodedAuthToken = "nobody:$AuthToken"
+  $unencodedAuthTokenBytes = [System.Text.Encoding]::UTF8.GetBytes($unencodedAuthToken)
+  $encodedAuthToken = [System.Convert]::ToBase64String($unencodedAuthTokenBytes)
+
+  if ($isDevOpsRun) {
```

Comment:
Are we sure this variable will be recognized? I know that scoping in PowerShell is funky, so not sure if a body of function definition will see a variable from dotsourced file.

Reply:
We define that globally in our logging.ps1 at https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/logging.ps1#L1. And that is typically how we determine if we can call the DevOps specific logging commands. This variable will be defined anytime our logging functions are also defined otherwise, we will have errors.